fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1251,20 +1250,9 @@ public void testGetObjectGraphWithChildren() throws IOException {
         httpGet.setHeader("Limit", Integer.toString(CHILDREN_LIMIT));
         try (final CloseableHttpResponse response = execute(httpGet)) {
             try (final CloseableGraphStore graph = getGraphStore(response)) {
-                assertTrue("Should contain child count!",
-                        graph.contains(ANY, createURI(location), HAS_CHILD_COUNT.asNode(), ANY));
-
-                final Iterator<Quad> children = graph.find(ANY, createURI(location), HAS_CHILD_COUNT.asNode(), ANY);
-                assertTrue("Should have a child count triple!", children.hasNext());
-
-                // Total child count should be provided
-                final Node child = children.next().getObject();
-                assertTrue("Object should be a literal!", child.isLiteral());
-                assertEquals(CHILDREN_TOTAL, Integer.parseInt(child.getLiteralValue().toString()));
-
                 final Iterator<Quad> contains = graph.find(ANY, createURI(location), CONTAINS.asNode(), ANY);
                 assertTrue("Should find contained child!", contains.hasNext());
-                assertEquals(CHILDREN_LIMIT - 1, Iterators.size(contains));
+                assertEquals(CHILDREN_LIMIT, Iterators.size(contains));
```

Comment:
I'm am good with merging this. I have pulled it and tested it, no issues building.

I am just wondering how come this test is changing with this PR? I must be missing something.